mlflow/gateway/client.py

```diff
@@ -33,7 +35,8 @@ class MlflowGatewayClient:
     def __init__(self, gateway_uri: Optional[str] = None):
         self._gateway_uri = gateway_uri or get_gateway_uri()
         self._host_creds = self._resolve_host_creds()
-        self._route_base = MLFLOW_GATEWAY_ROUTE_BASE
+        self._crud_route_base = MLFLOW_GATEWAY_CRUD_ROUTE_BASE
+        self._query_route_base = MLFLOW_GATEWAY_ROUTE_BASE
```

Comment:
Do we nees these attributes? Can we directly use `MLFLOW_GATEWAY_CRUD_ROUTE_BASE` and `MLFLOW_GATEWAY_ROUTE_BASE`?

Reply:
good point. removing